yb-voyager/src/srcdb/data/gather-assessment-metadata/oracle/table-index-sizes.sqlplus

```diff
@@ -0,0 +1,57 @@
+SET ECHO OFF
+SET FEEDBACK OFF
+SET HEADING ON
+SET MARKUP CSV ON DELIMITER , QUOTE OFF
+SET PAGESIZE 0
+SET TRIMSPOOL ON
+SET TERMOUT OFF
+SET VERIFY OFF
+
+-- Get the session ID
+COLUMN session_id NEW_VALUE session_id
+SELECT SYS_CONTEXT('USERENV', 'SESSIONID') AS session_id FROM dual;
+
+-- Create a temporary table to hold the data
+CREATE GLOBAL TEMPORARY TABLE temp_table_index_sizes_&session_id
+ON COMMIT PRESERVE ROWS AS
+    SELECT
+        owner AS schema_name,
+        segment_name AS object_name,
+        'table' AS object_type,
+        bytes AS size_in_bytes
+    FROM
+        dba_segments
+    WHERE
+        segment_type IN ('TABLE', 'TABLE PARTITION', 'TABLE SUBPARTITION')
```

Comment:
I think there are other segment types here, including `INDEX`/`INDEX PARTITION`, so how about using them directly instead of doing UNION for indexes, or are they not complete?

Reply:
there are some category of indexes which are unsupported(Cluster, IOT...) and if i just include the segments(INDEX and INDEX PARITTION) then it will contain those also. So thought to keep them separate.

Reply:
similar kind of change is just pushed to `object-type-mapping.sqlplus`

Reply:
Okay makes sense.